docs/cli.md

```diff
@@ -29,8 +34,281 @@ Common Flags
+|`--context-dir` *dir*         | Use *dir* as context dir in the build              |
+|`--image-stream` (`-i`) *is*  | Use imagestream *is* in the app                    |
+|`--docker-image` *image*      | Include Docker image *image* in the app            |
+|`--template` *t*              | Use OpenShift stored template *t* in the app       |
+|`--file` *filename*           | Use template in *filename*                         |
+|`--param` (`-p`) *k1=v1,...*  | Set/override parameters *k1...* with *v1...*       |
+|`--group` *comp1*`+`*comp2*   | Group together components *comp1* and *comp2*      |
+|`--env` (`-e`) *k1=v1,...*    | Set env vars *k1...* to values *v1...*             |
+|`--name` *name*               | Give *name* to all generated app artifacts         |
+|`--strategy` *s*              | Use build strategy *s*, one of: `docker`, `source` |
+|`--labels` (`-l`) *k1=v1,...* | Label all resources with *k1=v1,...*               |
+|`--insecure-registry`         | Bypass cert checks for referenced Docker images    |
+|`--output` (`-o`) *format*    | *format* is one of: `json`, `yaml`, `template`, `templatefile` |
+|`--output-version` *version*  | Output with *version* (default api-version)        |
+|`--no-headers`                | For default output, don't print headers            |
+|`--output-template` *s*       | Template string (`-o template`) or path (`-o templatefile`) |
+
+The template format is [golang templates](http://golang.org/pkg/text/template/#pkg-overview).
+The following example uses a MySQL image in a private registry to create an app and override application artifacts' names.
+
+```
+$ oc new-app \
+  --docker-image=myregistry.com/mycompany/mysql \
+  --name=private
+```
+
+See also [`oc start-build`](#oc-start-build).
+
+### oc status
+
+This shows a high level overview of the current project.
+
+See also [`oc describe`](#oc-describe) and [`oc get`](#oc-get).
+
+### oc project
+
+This displays the current project, or switches to another one.
+
+For example:
+
+```
+# Switch to the myapp project
+$ oc project myapp
+
+# Display the project currently in use
+$ oc project
+```
+
+## Build and Deploy Commands
+
+### oc start-build
+
+This command will manually start a build by specifying either a buildConfig or
+a build name with the `--from-build` flag. There is also the option of streaming
+the logs of the build if the `--follow` flag is specified.
+
+```
+$ oc start-build ruby-sample-build
+$ oc start-build --from-build=ruby-sample-build-275d3373-c252-11e4-bc79-080027c5bfa9
+$ oc start-build --from-build=ruby-sample-build-275d3373-c252-11e4-bc79-080027c5bfa9 --follow
+```
+
+See also [`oc new-build`](#oc-new-build) and [`oc new-app`](#oc-new-app).
+
+### oc build-logs
+
+> **NOTE**: This command will be later replaced by upstream (see: [kubectl log](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl_logs.md#kubectl-logs)).
```

Comment:
Please change kubectl log (deprecated) to kubectl logs. I had fixed that link before but forgot to change the actual command name.